.gitmodules

```diff
@@ -11,3 +11,6 @@
 [submodule "libs/tinyxml2"]
 	path = libs/tinyxml2
 	url = https://github.com/leethomason/tinyxml2.git
+[submodule "grpc/server/proto"]
```

Comment:
@JonasVautherin So we have now created DroneCore-Proto repo for maintaining `.proto` files. I couldn't figure out why it is done so. We can as well have this in `grpc/server/proto/` isn't it ?

Reply:
@JonasVautherin 
What I understand is that, we now have
- **DroneCore-Python**:  Repo that has Python-Client samples for DroneCore like `fly_mission.py`, `takeoff_land.py`, etc.

- **DroneCore-Proto**: Repo that contains `.proto` files alone to be used by all the language bindings.

We're going to have in future:
- **DroneCore-Android:** Repo that is going to have Android app samples for DroneCore like `FlyMission` app, etc.
Please feel free to correct me.

Reply:
You're perfectly right! The idea is to split the clients from the backend. And since we need to share the proto files between them, I figured having a repo used as a submodule would be nice.

I am working on DroneCore-Python and DroneCore-Java that I will push very soon to get your feedback :-).

Reply:
Okay makes sense to me.